DependencyInjection/Builder/RegistrationRepositoryBuilder.php

```diff
@@ -23,7 +23,7 @@
 namespace OAT\Bundle\Lti1p3Bundle\DependencyInjection\Builder;
 
 use InvalidArgumentException;
-use OAT\Bundle\Lti1p3Bundle\Registration\RegistrationRepository;
+use OAT\Bundle\Lti1p3Bundle\Repository\RegistrationRepository;
 use OAT\Library\Lti1p3Core\Platform\PlatformFactory;
 use OAT\Library\Lti1p3Core\Platform\PlatformInterface;
 use OAT\Library\Lti1p3Core\Registration\Registration;
```

Comment:
I know it's out of this PR scope, but I think you should use your RegistrationFactory on line 197 (sorry I cannot comment directly on the line)

Reply:
totally ! done